### PR TITLE
SG-21170-ROUND2 [ZD to AKN] Update links referenced from code to the old support site to point to AKN

### PR DESCRIPTION
**Pull request by 000paradox000**
This PR replaces ZD urls to AKN and developer urls

### CODE REVIEW

File: python/tank/descriptor/descriptor.py [hunk 1]
```diff
@@ -358,7 +358,7 @@ def support_url(self):
         meta = self._get_manifest()
         support_url = meta.get("support_url")
         if support_url is None:
-            support_url = "https://support.shotgunsoftware.com"
+            support_url = "https://developer.shotgridsoftware.com"
```
Comment (mathurf) [inline python/tank/descriptor/descriptor.py:361]: This one is tricky. That variable is used in different places and it is mostly mean to contact support. I think we might need to use this URL: https://help.autodesk.com/view/SGSUB/ENU/

What do you think?

File: python/tank/constants.py [hunk 2]
```diff
@@ -83,7 +83,7 @@
 TOOLKIT_INIT_CACHE_FILE = "toolkit_init.cache"
 
 # URL for contacting support
-SUPPORT_URL = "https://support.shotgunsoftware.com"
+SUPPORT_URL = "https://developer.shotgridsoftware.com"
```
Comment (mathurf) [inline python/tank/constants.py:86]: This one is tricky. That variable is used in different places and it is mostly mean to contact support. I think we might need to use this URL: https://help.autodesk.com/view/SGSUB/ENU/

What do you think?

**Review by mathurf — CHANGES_REQUESTED** [review]
Please fix the 2 URLs. Reach out to me if you don't agree with my comments

File: python/tank/constants.py (hunk 2 above)
Comment (000paradox000) [reply python/tank/constants.py]: https://knowledge.autodesk.com/contact-support

**Review by mathurf — APPROVED** [review]
LGTM!

**Pull request merged**